app/src/test/java/org/kiwix/kiwixmobile/zimManager/libraryView/adapter/LibraryListItemTest.kt

```diff
@@ -35,45 +32,45 @@ import org.kiwix.kiwixmobile.zimManager.libraryView.adapter.LibraryListItem.Book
 
 internal class LibraryListItemTest {
 
-  val book = mockk<Book>()
+  private val book = Book()
 
   @BeforeEach
   fun init() {
-    clearAllMocks()
-    every { book.getId() } returns "0"
-    every { book.getSize() } returns "0"
```

Comment:
Why are we removing this?

Reply:
hi @gouri-panda ,

i have removed this because ```mock``` is not working with ```@JvmField``` and without ```@JvmField``` it's gives error
```
Caused by: org.simpleframework.xml.core.ConstructorException: Default constructor can not accept read only @org.simpleframework.xml.ElementList(data=false, empty=true, entry=, inline=true, name=book, required=false, type=void) on method 'book$annotations' in class org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity
```
so now i removed the ```@JvmField``` and fix ```ConstructorException```.